framework/wazuh/core/exception.py

```diff
@@ -339,6 +339,8 @@ class WazuhException(Exception):
                },
         1759: {'message': 'Timeout acquiring client.keys lock, another thread or process might be blocking it',
                },
+        1760: {'message': 'Feature only available for older agent versions, it doesn\'t apply for more recent ones.'
```

Comment:
Perhaps we should specify up to what version of agents it is possible to use this endpoint, maybe as part of the remediation, with which we will also avoid this null message:

```JSON
      {
        "error": {
          "code": 1760,
          "message": "Feature only available for older agent versions, it doesn't apply for more recent ones.",
          "remediation": null
        },
```

Reply:
Solved in eb163306892b4b888cd6693a98eaf32abb16b160.